gt/main.c

```diff
@@ -1437,6 +1481,9 @@ process_gt_cmd(struct gt_cmd_entry *entry, struct gt_instance *instance)
 				rte_lcore_id());
 		}
 
+		if (entry->u.bc.is_returned)
+			return_message(instance);
```

Comment:
We need to pop out of the Lua stack the string once `return_message()` returns, otherwise we have a memory leak since Lua will never be able to free that string.